docs/scenarios/actions.rst

```diff
@@ -125,6 +125,9 @@ The following example is used to:
       </action>
     </recv>
 
+.. note::
+  Release 3.6.0 added XML syntax checks and the characters & and < are
+  not valid in XML files.  Replace those with &amp; &lt; in strings.
```

Comment:
I would say:
```
  Release 3.6.0 added rudimentary XML syntax checks. Now the &
  and < characters must be escaped even inside attribute values.
```

Reply:
Can we explain how to escape them?  Again I expect a lot of people to trip over this and many of them are not programmers and may have only ever used XML with SIPp.  e.g. my entire SVT test organization.

  Release 3.6.0 added rudimentary XML syntax checks. Now the &
  and < characters must be escaped (&amp; &lt;) even inside attribute values.

Reply:
We can. How about:
```
Release 3.6.0 added rudimentary XML syntax checks. Now the & and <
characters must be escaped as &amp; and &lt; even inside attribute values.
```